chore(release): Automated agent release notes (v0.6.5) (#910)
armory/docs: content/en/docs/release-notes/rn-armory-agent/agent-service/agent-service-v-0-6-5.md

---
title: v0.6.5 Armory Agent Service (2021-09-30)
toc_hide: true
version: 00.06.05

---

### New

> - Resolve resource mapping for non discovered kinds during agent startup, this allows operations to complete even if there was a failure when reading a specific kind during startup provided the agent has permissions to read it.

Reviewer: ```suggestion
- Resolve resource mapping for non discovered kinds during agent startup. This allows operations to complete even if there is a failure when reading a specific kind during startup as long as the agent has permissions to read it.
```
